src/container/cli.c

```diff
@@ -2556,6 +2593,8 @@ dc_cont_get_attr(tse_task_t *task)
 
 out:
 	tse_task_complete(task, rc);
+	if (new_names != NULL)
+		D_FREE(new_names);
```

Comment:
As a general rule we don't check for NULL before free.
```suggestion
	D_FREE(new_names);
```